hw1_mzw7af_vl4kz.py

```diff
@@ -57,6 +57,30 @@ def main():
     facts = {}
     rules = []
 
+    for line in sys.stdin:
+        argArray = line.strip().split(" ")
 
+        if argArray[0].lower() == "teach":
+            if argArray[3] == "=":
+                stringVar = ' '.join(argArray).split(" = ")[1]
+                teachVar(argArray[1], argArray[2], stringVar)
+                #print(argArray[1] + " " + argArray[2] + " " + stringVar)
+            elif argArray[2] == "=":
+                teachRootVar(argArray[1], argArray[len(argArray)-1])
```

Comment:
why the whole `argArray[len(argArray)-1]` business? Can't you just do argArray[3] ?